javascript/javascript1/week2/flight-booking-fullname-function.js

```diff
@@ -0,0 +1,24 @@
+const male = 'Lord';
+const female = 'Lady';
+
+function getFullname(firstname, surname, useFormalName, gender) {
+  if (
+    (typeof firstname !== 'string' && typeof surname !== 'string') ||
+    firstname === '' ||
+    surname === ''
+  ) {
+    console.log('Please use letters to enter the name');
+  }
+  if (useFormalName && gender === male) {
+    return `${male} ${firstname} ${surname}`;
+  } else if (useFormalName && gender === female) {
+    return `${female} ${firstname} ${surname}`;
+  }
+  return `${firstname} ${surname}`;
+}
+
+const fullname1 = getFullname('Yuliia', 'Balandiuk', true, female);
```

Comment:
It would be strange to use a variable as input. Maybe "female"? And then you can check based on gender should it return `Lady` or `Lord`